source/iface/properties.rst

```diff
@@ -2,20 +2,191 @@
   Copyright 2020 The Khronos Group Inc.
   SPDX-License-Identifier: CC-BY-4.0
 
-.. _property_list:
+.. _properties:
 
+**********
+Properties
+**********
+
+Each of the following SYCL runtime classes: ``accessor``, ``buffer``,
```

Comment:
I propose to make it with refs (like `` :ref:`accessor` ``, `` :ref:`buffer` `` etc. ) like I did in the By-value and Reference semantics (approach is approved by Mike), for those types without refs we can just leave it like `` `accessor` `` and update it later, on cycle 2, when everything will be implemented. (I left comments in my files as a reminder)

Reply:
this could be part 2 of work...currently we have no all reff e.g accessor

Reply:
Yup, I agree. Maybe you could leave a comment here to not miss this place in cycle 2. F.e. like I did in the "Reference Semantics"